go-controller/pkg/ovn/controller/services/utils.go

```diff
@@ -3,95 +3,40 @@ package services
 import (
 	"fmt"
 	"net"
-	"strconv"
+	"strings"
 
 	"github.com/pkg/errors"
 
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
-	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/ovn/acl"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/ovn/gateway"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/ovn/loadbalancer"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
 	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
 
 	v1 "k8s.io/api/core/v1"
-	discovery "k8s.io/api/discovery/v1beta1"
 	"k8s.io/apimachinery/pkg/util/sets"
-
 	"k8s.io/klog/v2"
 )
 
-type lbEndpoints struct {
-	IPs  []string
-	Port int32
-}
-
-// return the endpoints that belong to the IPFamily as a slice of IPs
-func getLbEndpoints(slices []*discovery.EndpointSlice, svcPort v1.ServicePort, family v1.IPFamily) lbEndpoints {
-	epsSet := sets.NewString()
-	lbEps := lbEndpoints{[]string{}, 0}
-	// return an empty object so the caller don't have to check for nil and can use it as an iterator
-	if len(slices) == 0 {
-		return lbEps
+func deleteVIPsFromOVN(vips sets.String, st *serviceTracker, name, namespace string) error {
```

Comment:
st *serviceTracker is unused param here, remove it

Reply:
also change the name of this function to something more obvious like: deleteVIPsFromAllLoadBalancers or something to indicate we are removing VIPs from every lb, not just cluster and worker